rts/Lua/LuaVAOImpl.cpp

```diff
@@ -164,13 +163,12 @@ int LuaVAOImpl::AddObjectsToSubmissionImpl(const sol::stack_table& ids)
 {
 	const std::size_t idsSize = ids.size(); //size() is very costly to do in the loop
 
-	DrawCheckInput inputs = {
-		std::nullopt,                // drawCount
-		std::nullopt,	             // baseVertex
-		std::nullopt,	             // baseIndex
-		submitCmds.size() + idsSize, // instCount
-		std::nullopt,                // baseInstance
-	};
+	DrawCheckInput inputs = {};
+	inputs.drawCount    = std::nullopt;
+	inputs.baseVertex   = std::nullopt;
+	inputs.baseIndex    = std::nullopt;
+	inputs.instCount    = submitCmds.size() + idsSize;
+	inputs.baseInstance = std::nullopt;
```

Comment:
perhaps there should be a constructor that takes these 5 args?

Reply:
Yes, I don't know why the previous code stopped working on 17->20 transition, I'm not proud with the fix, so any suggestions are welcome.